truffle/com.oracle.truffle.api.instrumentation/src/com/oracle/truffle/api/instrumentation/SourceSectionFilter.java

```diff
@@ -362,7 +362,7 @@ public SourceSectionFilter build() {
         }
 
         private void verifyNotNull(Object[] values) {
-            if (values == null) {
+            if (values == null || values.length == 0) {
                 throw new IllegalArgumentException("Given arguments must not be null.");
```

Comment:
Yeah, and tests fail because of this change. So, for me the question is, is this here the intended behavior, or would there be a reason to allow for empty arrays?